app/Providers/RouteServiceProvider.php

```diff
@@ -41,7 +41,7 @@ public function boot()
 
             $class = HandleBindingRepository::detectResourceType();
 
-            return HandleBindingRepository::bindResourceInstance($class, $value);
+            return HandleBindingRepository::bindResourceModelInstance($class, $value);
```

Comment:
HandleBindingRepository was refactored, so, now bindResourceInstance resolve a full class, without prefix. a new bindResourceModelInstance was added to fetch models instances located on "App" folder.